freshmaker/kojiservice.py

```diff
@@ -37,11 +37,16 @@
 import re
 import requests
 import yaml
-import freshmaker.utils
-from freshmaker import log, conf, db
-from freshmaker.consumer import work_queue_put
-from freshmaker.events import BrewContainerTaskStateChangeEvent
-from freshmaker.models import ArtifactBuild
+
+import gi
+gi.require_version('Modulemd', '2.0')  # noqa E402
```

Comment:
Any `import` that goes under this, needs `# noqa E402`. :( Can we perhaps hide this under separate module (e.g. `freshmaker.modulemd`)?

Reply:
Yes. But it seems sth come be better, change it a bit. 
I think this is a package we will use, not wrapping a new service, so I prefer not to put it in a separate module.

I tried to put import gi, gi.require_version('Modulemd', '2.0')  # noqa E402 at the end of import chain and there is no flake8 issue, but I guess there is some convention to put the freshmaker model at the end.

Reply:
The idea was to create simple file `freshmaker/modulemd.py`:

```python
import gi
gi.require_version('Modulemd', '2.0')
from gi.repository import Modulemd  # noqa E402
```

And use `from freshmaker.modulemd import Modulemd`.

Reply:
Yeah. I agree that this will resolve the flake issues. But do we have strong opinion to create a new file for fixing flake8 issues, is this a convention for fixing this kind of issues, I am not sure.